Asterisk.2013/Asterisk.NET/IO/SocketConnection.cs

```diff
@@ -156,7 +154,8 @@ public string ReadLine()
 		/// <summary>connection has already been closed.</summary>
 		public void Write(string s)
 		{
-			writer.Write(s);
+			writer.Write(encoding.GetBytes(s));
+            writer.Flush();

```

Comment:
One is this the only line with spaces instead of tabs?
Not a big issue, just makes reading the code on GitHub weird.

Reply:
Visual Studio Text Editor. I have nothing to do with =)

Reply:
I know. There are other places in this project where this is an issue, and it is really just annoying to read. Other than that, it really doesn't matter. ^^;